README.md

```diff
@@ -16,11 +16,16 @@ Deploy [Prometheus](https://github.com/prometheus/prometheus) monitoring system
 
 When upgrading from <= 2.4.0 version of this role to >= 2.4.1 please turn off your prometheus instance. More in [2.4.1 release notes](https://github.com/cloudalchemy/ansible-prometheus/releases/tag/2.4.1)
 
+### Install with docker
```

Comment:
For readability reasons this section should be mentioned after "Role variables".